controllers/gitopsservice/gitopsservice_controller.go

```diff
@@ -189,7 +167,7 @@ func (r *ReconcileGitopsService) Reconcile(request reconcile.Request) (reconcile
 
 	// Fetch the GitopsService instance
 	instance := &pipelinesv1alpha1.GitopsService{}
-	err := r.client.Get(context.TODO(), types.NamespacedName{Name: serviceName}, instance)
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: serviceName}, instance)
```

Comment:
When the new `Reconcile(context.Context, reconciler.Request) (reconciler.Result, error)` method is implemented, the calls to `context.TODO()` should be changed to use the `ctx` parameter passed in.